compose/foundation/foundation/src/skikoTest/kotlin/androidx/compose/foundation/copyPasteAndroidTests/FocusableTest.kt

```diff
@@ -330,7 +332,9 @@ class FocusableTest {
 
     @Test
     fun focusable_inspectorValue() = runSkikoComposeUiTest {
-        val modifier = Modifier.focusable() as InspectableValue
+        val modifier = Modifier.focusable()
+            .foldIn(mutableListOf<Modifier.Element>()) { acc, e -> acc.apply { acc.add(e) } }
```

Comment:
```suggestion
            .foldIn(mutableListOf<Modifier.Element>()) { acc, e -> acc.apply { add(e) } }
```

Reply:
Thanks, I just copied this part from androidx, but there was util finction, so I think that's ok to change it